src/storage/slab/slab.h

```diff
@@ -105,6 +107,7 @@ struct slab {
     rel_time_t        utime;        /* last update time in secs */
     uint8_t           id;           /* slabclass id */
     uint32_t          padding:24;   /* unused */
+    uint32_t          refcount;     /* number of reserved items */
```

Comment:
updated in the next commit, please see if it's better

" # items that can't be evicted"